<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigTabShortcuts</class>
 <widget class="QWidget" name="ConfigTabShortcuts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>429</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>A&amp;pplication Shortcuts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="ShortcutsWidget" name="shortcutsWidgetGeneral" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSystemWide">
      <attribute name="title">
       <string>S&amp;ystem Shortcuts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ShortcutsWidget" name="shortcutsWidgetSystemWide" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutsWidget</class>
   <extends>QWidget</extends>
   <header>gui/shortcutswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
